Dockerfile

```diff
@@ -1 +0,0 @@
-FROM appcelerator/node:amp
```

Comment:
I just merged the PR before realizing that I think you need to update this to:
`FROM appcelerator/node:amp-onbuild`. I'm going to reopen the related issue.